.github/workflows/main.yml

```
@@ -39,6 +39,8 @@ jobs:
         run: just sacro-app/build
 
   test-windows:
```

This step isn't just testing any more, so might be worth renaming it for clarity. This is just a suggestion:

```suggestion
  build-windows:
```